workers/bioinformatics_mill_workers/celery.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, unicode_literals
+import os
+from celery import Celery
```

Comment:
That's a good point. I am definitely not in favor of leaving this file named celery. It was named so in an example I was following and I forgot to change it.